rust/protocol/src/sealed_sender.rs

```diff
@@ -821,14 +943,18 @@ mod sealed_sender_v2 {
         }
     }
 
+    /// Encrypt or decrypt a slice of random bytes `input` using a shared secret derived from
+    /// `our_keys` and `their_key`.
+    ///
+    /// The output of this method when called with [`Direction::Sending`] can be inverted to produce
+    /// the original `input` bytes if called with [`Direction::Receiving`] with `our_keys` and
+    /// `their_key` swapped.
```

Comment:
I like these simple descriptions a lot better, thank you.

Reply:
:DDDD